src/parser/parser.ts

```diff
@@ -97,6 +97,11 @@ export interface Method extends Member {
 
 	statements: Statement[];
 
+	/**
+	 * One line before the method seperator
+	 */
+	oneLineB4: number
```

Comment:
At first I thought it is special meaning of B4 somewhere.. But now I think it's like the name: B4 - before. Is it correct? It was not obvious at first look :(

Reply:
I agree the name is confusing. I also think this may not be valuable as a property.